### ISSUE TITLE
GetAccountBalance() Crash

### ISSUE THREAD

**Issue by popey456963** [Bara20/TroubleinTerroristTown#53]
Seem to have an odd crash that started when I updated to the latest version of the TTT to do with GetAccountBalance().  The crash log can be found https://crash.limetech.org/4ci7t2x3lj4s.  This crash does not seem unique, it's happened 7 times in the past day and seems to be more prevalent during the first 10 seconds of a map.  Plugin List:

```
 "Web Shortcuts CS:GO version" (2.4) by Franc1sco franug and James "sslice" Gray
 "Basic Comm Control" (1.8.0.5919) by AlliedModders LLC
 "Store - Trade System" (1.0) by Zephyrus
 "iNilo Chat Controller" (0.0.548) by iNilo.net
 "CS:GO Movement Unlocker" (1.0) by Peace-Maker
 "iNilo TTT" (0.0.1) by iNilo.net
 "iNilo Sprays" (0.0.584) by iNilo.net
 "Fix DeadTalk" (1.0.0) by Bara
 "Advertisements" (2.0.2) by Tsunami
 "AFK Manager Lite - CSS" (1.1-css) by KorDen, orig by Rothgar
 "Updater" (1.2.2) by GoD-Tony
 "No Weapon Fix" (1.3.1) by .#Zipcore
 "Nextmap" (1.8.0.5919) by AlliedModders LLC
 "Sound Commands" (1.8.0.5919) by AlliedModders LLC
 "Basic Commands" (1.8.0.5919) by AlliedModders LLC
 "Fun Commands" (1.8.0.5919) by AlliedModders LLC
 "Admin Help" (1.8.0.5919) by AlliedModders LLC
 "iNilo Chat Processor" (0.0.261) by iNilo.net
 "SourceBans: Admin Config Loader" ((SB++) 1.5.4.6) by AlliedModders LLC, Sarabveer(VEER™)
 "Anti-Flood" (1.8.0.5919) by AlliedModders LLC
 "Admin File Reader" (1.8.0.5919) by AlliedModders LLC
 "SourceMod Radio" (1.0.0.22) by dubbeh
 "SourceSleuth" ((SB++) 1.5.4.6) by ecca, Sarabveer(VEER™)
 "Player Commands" (1.8.0.5919) by AlliedModders LLC
 "Custom Player Skins (Core)" (1.3.4) by Mitchell, Root
 "iNilo's name cleaner" (0.0.265) by iNilo.net
 "Basic Votes" (1.8.0.5919) by AlliedModders LLC
 "Store - The Resurrection" (1.0) by Zephyrus
 "Map Nominations Extended" (1.10.0) by Powerlord and AlliedModders LLC
 "Connect MSG" (1.0) by Crazy
 "Basic Info Triggers" (1.8.0.5919) by AlliedModders LLC
 "Admin Menu" (1.8.0.5919) by AlliedModders LLC
 "iNilo Chat Basics" (0.0.514) by iNilo.net
 "Reserved Slots" (1.8.0.5919) by AlliedModders LLC
 "SourceBans++" (1.5.4.2) by SourceBans Development Team, Sarabveer(VEER™)
 "MapChooser Extended" (1.10.2) by Powerlord, Zuko, and AlliedModders LLC
 "Thirdperson Mode" (1.0) by Zephyrus
 "Client Preferences" (1.8.0.5919) by AlliedModders LLC
 "Fun Votes" (1.8.0.5919) by AlliedModders LLC
 "Hide Radar" (1.0.0) by Bara
 "iNilo Modules" (0.0.1) by iNilo.net
 "TTT - Items: Taser" (2.1.<ID>) by Bara, .#Zipcore, Darkness, whocodes & good_live
 "DC BLOCKER" (1) by Niko
 "Rock The Vote Extended" (1.10.0) by Powerlord and AlliedModders LLC
 "SM Grabber mod" (1.0) by Franc1sco Steam: franug
 "SourceBans Checker" ((SB++) 1.5.4.6) by psychonic, Ca$h Munny, Sarabveer(VEER™)
 "crashcatcher.smx" crashcatcher.smx
 "TTT - Icons" (2.1.B153977644) by Bara, .#Zipcore, Darkness, whocodes & good_live
 "TTT - Traitor Doors" (2.1.B153977644) by zipcore
 "TTT - Items: Healthshot" (2.1.B153977644) by Bara, .#Zipcore, Darkness, whocodes & good_live
 "TTT - Overlays" (2.1.B153977644) by Bara, .#Zipcore, Darkness, whocodes & good_live
 "TTT - Shop" (2.1.B153977644) by Bara, .#Zipcore, Darkness, whocodes & good_live
 "TTT - Items: Decoy Teleporter" (2.1.B153977644) by Bara, .#Zipcore, Darkness, whocodes & good_live
 "TTT - Trouble in Terrorist Town" (2.1.<ID>) by Bara, .#Zipcore, Darkness, whocodes & good_live
 "TTT - Items: ID" (2.1.B153977644) by Bara, .#Zipcore, Darkness, whocodes & good_live
 "TTT - SQL Manager" (2.1.B153977644) by Bara, .#Zipcore, Darkness, whocodes & good_live
 "TTT - Items: Fake Body" (2.1.B153977644) by Bara, .#Zipcore, Darkness, whocodes & good_live
 "TTT - Items: Weapons" (2.1.B153977644) by Bara, .#Zipcore, Darkness, whocodes & good_live
 "TTT - Spec Menu" (2.1.B153977644) by Bara, .#Zipcore, Darkness, whocodes & good_live
 "TTT - Items: Wallhack" (2.1.B153977644) by Bara, .#Zipcore, Darkness, whocodes & good_live
 "TTT - Hud" (2.1.B153977644) by Bara, .#Zipcore, Darkness, whocodes & good_live
 "TTT - Talk Override" (2.1.B153977644) by Bara, .#Zipcore, Darkness, whocodes & good_live
 "TTT - Glow" (2.1.B153977644) by Bara & zipcore
 "TTT - Crash Catcher" (2.1.B153977644) by Bara, .#Zipcore, Darkness, whocodes & good_live
 "TTT - Health/Hurt Stations" (2.1.B153977644) by Bara, .#Zipcore, Darkness, whocodes & good_live
 "TTT - Grabber Mod" (2.1.B153977644) by Bara, .#Zipcore, Darkness, whocodes & good_live
 "TTT - C4 & Jihad" (2.1.B153977644) by Bara, .#Zipcore, Darkness, whocodes & good_live
 "TTT - Items: Ice Knife" (2.1.<ID>) by Bara, .#Zipcore, Darkness, whocodes & good_live
 "TTT - TA-Grenade" (2.1.B153977644) by Bara, zipcore & Neuro Toxin
 "TTT - Items: Taser" (2.1.<ID>) by Bara, .#Zipcore, Darkness, whocodes & good_live
 "TTT - Dna Scanner" (2.1.B153977644) by Bara, .#Zipcore, Darkness, whocodes & good_live
 "TTT - Hide Radar" (2.1.B153977644) by Bara, .#Zipcore, Darkness, whocodes & good_live
 "SourceComms" ((SB++) 1.5.4.6) by Alex, Sarabveer(VEER™)
```

**Comment by Bara20** [comment]
Ask iNilo? ;)
>  "iNilo TTT" (0.0.1) by iNilo.net

**Comment by Bara20** [comment]
https://forums.alliedmods.net/showthread.php?p=2428655

**Comment by popey456963** [comment]
I'm pretty sure this version is identical to the current main branch, the only reason for the rename was to keep track of things that this was the iNilo branch (before you merged it in).

And, as far as I can tell, that second thread fixed it.  Upsetting it doesn't seem to show the error but thanks so much for the fix.  Must have been when I updated server, which would be exactly the same time as when I updated TTT.
